packages/commonwealth/client/scripts/views/modals/manage_contract_template_modal.tsx

```diff
@@ -58,12 +59,42 @@ export class ManageContractTemplateModal extends ClassComponent {
     m.route.set(`/${scope}/new/contract_template`);
   }
 
-  handleConfirm(e) {
+  async handleConfirm(e) {
     e.preventDefault();
-    $(e.target).trigger('modalcomplete');
-    setTimeout(() => {
-      $(e.target).trigger('modalexit');
-    }, 0);
+
+    const communityId = app.activeChainId();
+    const { slug, displayOption, nickname, displayName } = this.form;
+
+    const communityTemplate = {
+      community_id: communityId,
+      // TODO do we know template id if we are about to create it
```

Comment:
we do know the template id because template != CCT. The template will have already been used to populate the "template" dropdown.